enos/provider/g5k.py

```diff
@@ -102,7 +116,8 @@ def init(self, config, force_deploy=False):
 
         return (roles,
                 map(str, vip_addresses),
-                [network_interface, external_interface])
+                [network_interface, external_interface],
+                provider_network)
```

Comment:
Nice. Can you update the documentation of [provider.py](https://github.com/BeyondTheClouds/enos/blob/64628a6012bcdbb0e3a2ee1f8270dc188bbcc23d/enos/provider/provider.py#L13) and explain what is the purpose of this new parameter.